server/index.html

```diff
@@ -136,6 +598,21 @@ <h1 style="width:100%; text-align: center; color: white; font-size: 22px; font-w
                                 $(".connectionLabel").html(userList);
                                       }
                             });
+                                $(function() {
```

Comment:
$(document).ready(function{}); is the same as $(function(){});. There shouldn't be a ready event inside the ready event.